platform/entanglement/test/services/CopyServiceSpec.js

```diff
@@ -266,6 +285,20 @@ define(
                     });
                 });
 
+                it("throws an expection when performed on invalid inputs", function () {
```

Comment:
This spec only passes because the two describe blocks above also run.  It should have it's own setup block to ensure object and newParent are valid objects.